.github/workflows/docker.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and publish Docker images
+
+on:
+  push:
+    branches:
+      - '**'
+    tags:
+      - '*.*.*'
+    pull_request:
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
+        with:
+          images: ${{ secrets.DOCKERHUB_USERNAME }}/galactus
```

Comment:
In our specific case, this probably shouldn't be the USERNAME; Galactus is under the automuteus organization, but we don't have an automuteus Dockerhub user (I think we actually can't have a dockerhub username that matches the name of the org). I think `automuteus/galactus` would work in this case

Reply:
That's a good point. Didn't think about that since I never used organizations on Docker Hub. The reason I choose a secret here is that hardcoding the tag would mean the action will fail when someone who doesn't have access to your Docker repo decides to publish his fork (which also means I wouldn't have been to test it btw).

Reply:
My idea would be to use another secret to solve that.

Reply:
Yeah that sounds great, too :)

Reply:
The action is running on my branch atm, will mark this as resolved once I can verify everything still works.

Reply:
Worked 👍